src/daemon/remote.proto

```diff
@@ -5,8 +5,9 @@ package dist_clang.daemon.proto;
 
 // Sent from emitter to absorber.
 message Remote {
-  optional base.proto.Flags flags = 1;
-  optional string source          = 2;
+  optional base.proto.Flags flags        = 1;
+  optional string source                 = 2;
+  optional string sanitize_blacklist     = 3;
```

Comment:
I don't remember what version of protobuf library right now, but you should try to use the `Map` field (https://developers.google.com/protocol-buffers/docs/proto#maps). If the protobuf is too old, then leave a `FIXME` with a future plans to update protobuf and use fair map here.